### ISSUE TITLE
'5w unassigned' and '9w unassigned'

### ISSUE THREAD

**Issue by kitscorp** [kasperlab/Joost_et_al_2020_Cell_Stem_Cell#4]
I'm currently working on a project with your data set. I noticed that a few barcodes are assigned to multiple stages (e.g. both 5w2 and 9w1) and/or different replicates (e.g. both 5wk1 and 5wk2)  in `assigned_barcodes.txt`. Looking through your Jupyter notebooks, I saw that you labeled these as '5w unassigned' or '9w unassigned' during your workflow and kept them in the data set as such.

Could you explain what the reason is for these multiple assignments? Would there be a reason to exclude these for downstream analyses, or can these cells be treated as belonging to both?

**Comment by sim-joo** [comment]
These ambiguous assignments are a result of the rather complicated and awkward way our bioinformatics core cleaned up and collated the raw data (this was back in mid-2016 when the CellRanger pipeline was still rather new and less developed). 

Barcodes shared _between_ the 5wk and 9wk dataset clearly represent different cells as both timepoints were processed completely independent. 

As for ambiguous assignments _within_ each time point: we were pondering whether to remove them _a priori_. However, we noticed that these barcodes did not look like doublets or artefacts and decided to leave the in the data. We also noticed that whether or not they are excluded does not affect the downstream analysis in any significant way. But if you feel uncomfortable with these ambiguous assignments, feel free to kick them out.
